src/TurbulenceConvection/turbulence_functions.jl

```diff
@@ -1,10 +1,3 @@
-# convective velocity scale
-function get_wstar(bflux::FT) where {FT}
```

Comment:
We will most likely need w_star also in entrainment/detrainment closure. Lets not delete this function, please :)

https://github.com/CliMA/ClimaAtmos.jl/blob/f6ac83d54b1a06d8c3219ca8c8e6e7d700344026/src/tendencies/edmfx_closures.jl#L71

Reply:
OK, I can put it back. I also changed the function call [here](https://github.com/CliMA/ClimaAtmos.jl/pull/1447/files#diff-894896115b56c40ac35bb30dd1ab4fab89341bfaf90c38ae50d1eedfd0b2fd5dR71), I'll change it back.